src/sidebar.js

```diff
@@ -36,6 +36,7 @@ module.exports = {
             },
           ],
         },
+        "getting-started/integrate-with-ory-cloud-through-webhooks",
```

Comment:
I don't think this should go into "get started", webhooks and integrations are usually more advanced topics you do after you have the first PoC. I think this should go to guides -> identity -> web hooks

Reply:
Not a problem, I can move it back to `kratos/hooks` to live with the two other documents we have on that topic. Truth be told in `kratos/hooks` this document doesn't make much sense IMO as it just re-packages existing content so for me it should either stay here (as per @kmherrmann request) or be removed altogether. WDYT?

Reply:
hm, ok I need to sleep over this - will msg tomorrow

Reply:
@aeneasr Did you arrive at any conclusions regarding this matter? :)

Reply:
My bad, let's just keep it as is. I would move this document to the top-level "guides" section https://www.ory.sh/docs/guides/ and give it a shorter name (see suggestion below)

Reply:
Done. Moved to guides/cusotmizing Ory and renamed it to customizing Ory Cloud - should make more sense now :)